sql/core/src/test/scala/org/apache/spark/sql/DatasetSuite.scala

```diff
@@ -1538,11 +1538,9 @@ class DatasetSuite extends QueryTest with SharedSparkSession {
     checkAnswer(df.sort("id"), expected)
     checkAnswer(df.sort(col("id")), expected)
     checkAnswer(df.sort($"id"), expected)
-    checkAnswer(df.sort('id), expected)
```

Comment:
Yes. This is a real removal of this. This rings me something. 
@srowen . I'm wondering if we need to have a test coverage for this syntax because this syntax is still supported in any way.

Reply:
Yes this one I removed because it's actually redundant. It is either exactly the same as `.sort("id")` or `.sort("$id")` depending on which implicit kicks in. That's already true in Scala 2.12.